mautic/mautic
# Fix "like" and "not like" on segment update

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #4227
| BC breaks? | N
| Deprecations? | N

[//]: # ( Required: )
#### Description:

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Make one segment, with visited URL like `your_value`
2. Make one segment, with visited URL not like `your_value`
3. run `mautic:segments:update`
4. check lead are not in proper segement

#### Steps to test this PR:
1. Apply this PR
2. run `mautic:segments:update`
3. check lead are in proper segment
